src/main/java/racing/model/CarName.java

```diff
@@ -1,11 +1,12 @@
-package racing;
+package racing.model;
 
 public class CarName {
+
     public static final int MAX_NAME_LENGTH = 5;
     private final String name;
 
     public CarName(String name) {
-        if (checkNameLengthExceed(name)){
+        if (checkNameLengthExceed(name)) {
             throw new IllegalArgumentException("차 이름은 5자 이하여야 합니다. input: " + name);
```

Comment:
```suggestion
            throw new IllegalArgumentException("차 이름은 5자 이하여야 합니다. input: " + name);
```
`MAX_NAME_LENGTH = 6`으로 변경되면 에러메시지도 같이 변경되기 쉬울까요? 
놓치기 쉬운 부분아닐까요?